tensorflow/python/kernel_tests/variables_test.py

```diff
@@ -370,6 +370,15 @@ def testInitializerFunctionDevicePlacement(self):
       for i in v2.initializer.inputs:
         self.assertEqual(expected_group_v2, i.op.colocation_groups())
 
+  def testLoad(self):
+    with self.test_session():
+      var = tf.Variable(np.random.randn(5, 5))
```

Comment:
No problem.

There is a chance that two random sequences will be the same but it is insignificant. Or am I missing something?